hostfactory/host_provider/src/cyclecloud_provider.py

```diff
@@ -1045,6 +1045,7 @@ def main(argv=sys.argv):  # pragma: no cover
         else:
             import traceback
             traceback.print_exc()
+        sys.exit(1) 
```

Comment:
why do we need an explicit exit(1) here?
seems like symphony's going to be unhappy with that (and we're skipping the finalizer)